Makefile

```diff
@@ -11,13 +11,43 @@ ifeq ($(USER), root)
 else
 	npm install
 endif
-	pip install "setuptools>=0.9.8"
+	pip install setuptools
 	# order matters here, base package must install first
 	pip install -e .
 	pip install -e "file://`pwd`#egg=lemur[dev]"
 	pip install -e "file://`pwd`#egg=lemur[tests]"
 	node_modules/.bin/gulp build
 	node_modules/.bin/gulp package --urlContextPath=$(urlContextPath)
+	tar --exclude='htmlcov' \
+		--exclude='cover' \
+		--exclude='build' \
+		--exclude='dist' \
+		--exclude='node_modules' \
+		--exclude='bower_components' \
+		--exclude='docs/html' \
+		--exclude='docs/doctrees' \
+		--exclude='lemur/static/dist' \
+		--exclude='lemur/static/app/vendor' \
+		--exclude='wheelhouse' \
+		--exclude='lemur/lib' \
+		--exclude='lemur/bin' \
+		--exclude='lemur/lib64' \
+		--exclude='lemur/include' \
+		--exclude='docs/_build' \
+		--exclude='lemur/tests/tmp' \
+		--exclude='docker/lemur.dev.tar.gz' \
+		--exclude='env' \
+		--exclude='venv' \
+		--exclude='ENV' \
+		--exclude='.vscode' \
+		--exclude='.git' \
+		--exclude='.tmp/lemur-dev.tar.gz' \
+		--exclude='docker/lemur-dev.tar.gz' \
+		--exclude='docs/_build' \
+		--exclude='lemur/tests/tmp' \
+		-cvzf ./.tmp/lemur-dev.tar.gz . 1>/dev/null 1>/dev/null
+	mv ./.tmp/lemur-dev.tar.gz ./docker/
```

Comment:
In the docker/docker-compose.yml file we can modify the build context and then copy in the directory/files from the root.  
see [this example](https://github.com/DataDog/lemur/compare/francesco.marabotto/FEI-2738-fixing-lemur-local-dev...DataDog:lemur:mitchc/modify-docker-compose-context?expand=1). Its functionally the same as what we are doing here by moving the archive to the /docker directory and unpacking the code in the dockerfile. 
Take a look and let me know what you think. The only difference is that it requires the user to run
```docker compose -f docker/docker-compose.yml up --build```
From the project root.

Reply:
Thanks, I'll try it

Reply:
I did this, but the tar version is way faster: I cannot filter folders with the COPY command and then doing a chown on the site-packages folder inside the venv takes forever. Are you OK in keeping the TAR? It works with MacOS now.